integration/cursors/helpers.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cursors
```

Comment:
I'm a bit concerned with the [package name](https://go.dev/blog/package-names) here. For this experiment of breaking down the integration test in multiple packages, I think a more appropriate name might be an int prefix + name (example: intcursor) (similar to strconv, etc.).

It's not critical for now, but I'd be worried if we had this with external packages.

[OpenTelemetry module](https://github.com/open-telemetry/opentelemetry-go)  and the [gRPC module](https://github.com/grpc/grpc-go/), does exactly that (and breaks down the code with too many packages, I'd argue), making it impossible for you to quickly use a package or to quickly know what a package is about (you also require using aliases all the time, when it should be an exception).